### ISSUE TITLE
Nice but…

### ISSUE THREAD

**Issue by gianklug** [RoadJDK/Energy-Star-Night-Bot-2021#2]
Nice, dass das mit den Captchas jetzt funktioniert. Du brauchst btw keine random timings bei jeder Frage - es werden am ende alle Antworten auf mal gesendet, weshalb du dort einfach ein random delay reinmachen kannst.

**Comment by RoadJDK** [comment]
Klar, ich persönlich mag es lieber so, dass bei jeder Frage ein random Delay eingebaut ist, falls jemand schneller an die Tickets kommen will kann das natürlich abändern. Wenn ich Zeit habe werde ich eine Variable einbauen, welche das steuern kann. ;)
